src/ui/components/Navigation/Core/NavigationBreadcrumbs/NavigationBreadcrumbs.scss

```diff
@@ -5,10 +5,10 @@
 
     &__wrapper {
         position: absolute;
-        left: 0;
-        right: 0;
-        top: 0;
-        bottom: 0;
+        inset: 0;
+
+        line-height: 28px;
```

Comment:
Let's move `28px` to constant and use it here plus for `.dl-core-navigation-breadcrumbs` height style. It is common setting